lib/fluent/command/fluentd.rb

```diff
@@ -81,6 +81,23 @@
   opts[:log_path] = s
 }
 
+ROTATE_AGE = %w(daily weekly monthly)
+op.on('--log-rotate-age AGE', 'generations to keep rotated log files') {|age|
+  if ROTATE_AGE.include?(age)
+    opts[:log_rotate_age] = age
+  else
+    begin
+      opts[:log_rotate_age] = Integer(age)
+    rescue TypeError
+      usage "log-rotate-age should be #{rotate_ages.join(', ')} or a number"
+    end
+  end
+}
+
+op.on('--log-rotate-size SIZE', 'sets the size to rotate log files') {|s|
```

Comment:
`BYTES` is better to show the value is an integer, not "size" (in fluentd's configuration file, e.g., 256k, 8m).